ast/ddl.go

```diff
@@ -1227,15 +1227,16 @@ const (
 	TokuDBRowFormatUncompressed
 )
 
-// OnDuplicateCreateTableSelectType is the option that handle unique key values in 'CREATE TABLE ... SELECT'.
+// OnDuplicateKeyHandlingType is the option that handle unique key values in 'CREATE TABLE ... SELECT' or `LOAD DATA`.
 // See https://dev.mysql.com/doc/refman/5.7/en/create-table-select.html
-type OnDuplicateCreateTableSelectType int
+// See https://dev.mysql.com/doc/refman/8.0/en/load-data.html
```

Comment:
Now, TiDB is compatible with mysql 5.7, could you change `https://dev.mysql.com/doc/refman/8.0/en/load-data.html` to `https://dev.mysql.com/doc/refman/5.7/en/load-data.html`, do you use the new features in 8.0?

Reply:
fixed

Reply:
I see many other links in this repo point to mysql 5.7, do I need it replace it all, or submit another PR for it?